Add note about Azer's GMT badge (#8606)
ppy/osu-wiki: wiki/People/The_Team/Global_Moderation_Team/en.md

| ::{ flag=HU }:: [[ Another ]](https://osu.ppy.sh/users/3416573) | Hungarian | Chat moderation |
| ::{ flag=CH }:: [\[ryuu\]](https://osu.ppy.sh/users/5698467) | Russian | Chat moderation |
| ::{ flag=US }:: [abraker](https://osu.ppy.sh/users/4635891) |  | Forum moderation |

> | ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) |  | Tournament management{^task-Azer] |

Reviewer: ```suggestion
| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) |  | Tournament management[^task-Azer] |
```

| `#vietnamese` |  | ::{ flag=VN }:: [My Angel Chino](https://osu.ppy.sh/users/20547597), ::{ flag=ID }:: [Sies](https://osu.ppy.sh/users/6491991) |

<!-- TODO: History section wanted (Chat operators/GreenBAT, etc, etc) -->

## Notes

> [^task-Azer]: Is a part of the GMT but they are not held to the same responsibilities.

Reviewer: taking from https://discord.com/channels/188630481301012481/218677502141399041/932333129664962560

```suggestion
[^task-Azer]: While not actually being part of the GMT itself, Azer is in the user group by technicality due to needing GMT-level permissions as a World Cup organiser.
```

> [^task-Azer]: While not actually being part of the GMT itself, Azer is in the user group by technicality due to needing GMT-level permissions as a World Cup organiser.

Reviewer 2: ```suggestion
[^task-Azer]: While not actually being part of the GMT itself, Azer is in the user group by technicality due to needing GMT-level permissions as a World Cup organiser.

```
